src/services/sdk.ts

```diff
@@ -187,15 +186,21 @@ const dispatchFeedItem = (
   blockNumber: number
 ) => {
   const decoder = new TextDecoder();
+
+  if (!content.published) throw new Error("timestamp is required");
+  // new Date(content.published).getTime()
+  const timestamp = Date.parse(content.published);
+
   dispatch(
     addFeedItem({
       fromAddress: decoder.decode((message.fromId as any) as Uint8Array),
       blockNumber: blockNumber,
       hash: decoder.decode((message.contentHash as any) as Uint8Array),
-      timestamp: new Date(content.published).getTime(), // TODO: really wrong
+      timestamp: timestamp,
       uri: decoder.decode((message.url as any) as Uint8Array),
       content: {
```

Comment:
Non-blocking: there's a [`createNote()` method](https://github.com/LibertyDSNP/sdk-ts/blob/main/src/core/activityContent/factories.ts#L191) in `sdk/core/activityContent` that could simplify this for you.